.github/workflows/build.yml

```diff
@@ -171,7 +171,7 @@ jobs:
         echo " "
         echo "---- simple_app_under_app_service/ setup ----"
         echo " "
-        ./run_example.sh simple_app_under_app_service setup
+        ./run_example.sh --no-make-clean simple_app_under_app_service setup
```

Comment:
We've already built the utilities etc. in previous step of `./run_example.sh application_service setup`.

So we can now skip re-building those using this new option, added in a previous change.